src/Microsoft.OData.Edm/Csdl/Serialization/EdmModelCsdlSerializationVisitor.cs

```diff
@@ -191,9 +196,11 @@ protected override void ProcessNavigationProperty(IEdmNavigationProperty element
         {
             this.BeginElement(element, this.schemaWriter.WriteNavigationPropertyElementHeader);
 
+            // From 4.0.1 spec says: None, meaning a DELETE request on a source entity with related entities will fail
+            // So, we should fix it.

```

Comment:
Not sure what you mean by "we should fix it"

Reply:
:disappointed: I can't remember what i mean it. 
Maybe i want to say that "OnDelete" has "None, Cascade, SetNull, SetDefault" values defined in 4.01. But, ODL now only process "Cascade" and "None". So we should fix it to support the others.

---
In reply to: [443061490](https://github.com/OData/odata.net/pull/1775#discussion_r443061490) [](ancestors = 443061490)